cloud/deployment/deployment.go

```diff
@@ -783,6 +783,29 @@ var GetDeployments = func(ws, org string, client astro.Client) ([]astro.Deployme
 	return deployments, nil
 }
 
+var CoreGetDeployments = func(ws, org, deploymentId string, coreClient astrocore.CoreClient) ([]astrocore.Deployment, error) {
```

Comment:
```suggestion
var CoreGetDeployment = func(ws, org, deploymentId string, coreClient astrocore.CoreClient) ([]astrocore.Deployment, error) {
```
nit: we are getting a single deployment